clang/test/SemaSYCL/sycl_kernel-host.cpp

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -fsyntax-only -verify %s
```

Comment:
Can we merge this test with existing `test/SemaSYCL/kernel-attribute.cpp` test?

Reply:
Done.  Please review.